index.html

```diff
@@ -192,8 +192,47 @@ <h3 class="about-card-title">Skills</h3>
                 </ul>
             </div>
         </section>
-        
-        
+        <section class="contact-form">
+            <h2 class="contact-box contact-text">
+                I'm always interested in hearing about new projects,
+                 so if you'd like to chat please get in touch.
+            </h2>
+            <form action="submit" class="contact-form">
+                <div class="name">
+                    <input type="text" maxlength="30"  placeholder="Full name" required>
+                </div>
+                <div class="email">
+                    <input type="email" name="email"  placeholder="Email address" required>
+                </div>
+                <div >
+                    <textarea class="message" name="textarea"  maxlength="500" required placeholder=" Enter text here"></textarea>
```

Comment:
- [ ] The space between each input element in your form is wider than the Figma design. Kindly adjust your design accordingly 🙏 
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/186231544-4ed09762-65e2-471e-8631-cd1b6a7290c8.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/186231466-b2d13ecf-4fa9-45b2-be4f-d5ec9b87f757.png)